python/lsst/ip/isr/ptcDataset.py

```diff
@@ -176,11 +179,12 @@ class PhotonTransferCurveDataset(IsrCalib):
     Version 1.3 adds the `noiseMatrix` and `noiseMatrixNoB` attributes.
     Version 1.4 adds the `auxValues` attribute.
     Version 1.5 adds the `covMatrixSideFullCovFit` attribute.
+    Version 1.6 adds the `noiseList` attribute.
     """
 
     _OBSTYPE = 'PTC'
     _SCHEMA = 'Gen3 Photon Transfer Curve'
-    _VERSION = 1.5
+    _VERSION = 1.6
```

Comment:
This may be 1.7 now.  I know there's another ticket in flight that has changed the version to 1.6: DM-42163.